internal/indexmanagement/reconcile_test.go

```diff
@@ -53,6 +54,22 @@ var _ = Describe("Index Management", func() {
 		name := fmt.Sprintf("%s-im-%s", cluster.Name, mapping.Name)
 		cronjob = newCronJob(cluster.Name, cluster.Namespace, name, "*/5 * * * *", "", selector, tolerations, []core.EnvVar{}, false)
 	})
+	Describe("#newCronJobSecurityContext", func() {
+		Context("when cronjob is created", func() {
+			It("should have the correct security contexts", func() {
+				Expect(pointer.BoolDeref(cronjob.Spec.JobTemplate.Spec.Template.Spec.SecurityContext.RunAsNonRoot, false)).To(Equal(true))
```

Comment:
why not sticking with one style `pointer.BoolEqual` here too?

Reply:
I think that would be confusing, since it would be: `Expect( ..BoolEqual..)To Equal`, (two equals back to back) instead of just `Expect(..BoolDeref..)To Equal`

Reply:
I'm changing it to comparing the entire SecurityContext just like @xperimental suggested